### ISSUE TITLE
Can we create template also from Excel instead of having it created by User ?

### ISSUE THREAD

**Issue by rajeshgrs** [rajeshgrs/dq-cli#6]
@jeevanjagadeesh, 
Can we create command template also from Excel instead of having it created by User in templates folder?

AS-IS in oieexportobjects.txt
oie exportObjects -dn <<DomainName>> -un <<UserName>> -pd <<Password>> -sdn <<SecurityDomain>> -rs <<RepositoryService>> -pn <<ProjectName>> -fp "<<ExportFilePath>>" -ow <<OverwriteExportFile>> -oo "<<OtherOptions>>" -cp "<<ControlFilePath>>" >>

The alternate command is
oie exportObjects -DomainName <<DomainName>> -UserName <<UserName>> -Password <<Password>> -SecurityDomain <<SecurityDomain>> -RepositoryService <<RepositoryService>> -ProjectName <<ProjectName>> -ExportFilePath "<<ExportFilePath>>" -OverwriteExportFile <<OverwriteExportFile>> -OtherOptions "<<OtherOptions>>" -ControlFilePath "<<ControlFilePath>>" 

As you see that command option name can also be same as placeholder value name, I think it should be possible to create the template directly from the code, instead user having it ready in templates folder. This was we can avoid any manual errors while creating a template.
Let me know your thoughts.

Thanks!!

**Comment by jeevanjagadeesh** [comment]
@rajeshgrs , I prefer to keep it as .txt, since its a template and is never going to change (mostly). Giving the flexibility to user in the form of xls, could cause some manual typo issues and would be difficult to debug and figure out , in case of any typo issue :smile:

**Comment by rajeshgrs** [comment]
Yes, I agree. May be an external utility to generate temple file from excel?

**Comment by rajeshgrs** [comment]
Solution for this Issue will be very helpful from now on as we tend to create templates for lot of plugins to test. 
May be we can send a different argument to dq_cli.js to instruct to create template(something like "node dq_cli.js -src oie -option createtemplate"), so the program will locate given plugin sheet "oie.xlsx" and traverse through all the worksheets and creates a template for each worksheet. 

I believe this should take the priority along with sendmail program :-)
